hilti/runtime/include/types/stream.h

```diff
@@ -105,24 +112,28 @@ class Chunk {
         _offset = other._offset;
         _data = std::move(other._data);
         _next = std::move(other._next);
-        _chain = std::move(other._chain);
+        _chain = other._chain;
```

Comment:
Similar question: what's the reason for this change?

Reply:
This fixes a clang-tidy warning about an unnecessary `move` since the array here is trivially copyable. I'll move this to its own patch.